From 807e92e3999a04cf90b8a03083f8146e629fe987 Mon Sep 17 00:00:00 2001
From: Patrick Rudolph <patrick.rudolph@9elements.com>
Date: Wed, 11 Oct 2023 08:13:49 +0200
Subject: [PATCH 10/10] configurations: sbp1: Adjust PVCORE_NIC lower crit

The PVCORE_NIC output voltage can go as low as 0.8V depending
on the individual chip. Adjust the lower critical limit to prevent
a false threshold warning.

Change-Id: I576437710bab421414fe1215f46444a55e919521
Signed-off-by: Patrick Rudolph <patrick.rudolph@9elements.com>
---
 configurations/sbp1_baseboard.json | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/configurations/sbp1_baseboard.json b/configurations/sbp1_baseboard.json
index 51018bb..199e4ac 100644
--- a/configurations/sbp1_baseboard.json
+++ b/configurations/sbp1_baseboard.json
@@ -601,7 +601,7 @@
                     "Label": "vout1",
                     "Name": "lower critical",
                     "Severity": 1,
-                    "Value": 0.85
+                    "Value": 0.76
                 },
                 {
                     "Direction": "greater than",
@@ -667,7 +667,7 @@
                     "Label": "vout1",
                     "Name": "lower critical",
                     "Severity": 1,
-                    "Value": 0.85
+                    "Value": 0.76
                 },
                 {
                     "Direction": "greater than",
-- 
2.41.0

